pipelines/_ingredients/LSSTCam-imSim/DRP.yaml

```diff
@@ -245,54 +245,59 @@ subsets:
 
       Includes extended-level diff_matched_analysis metrics, which are
       considered core for DC2 as they can't be run on other datasets yet.
-  faro_all:
-    subset:
-      # visit-level on single-frame products
-      - nsrcMeasVisit
-      - TE3
-      - TE4
-      # tract-level, matched-visit on single-frame products
-      - matchCatalogsTract
-      - matchCatalogsPatch
-      - matchCatalogsPatchMultiBand
-      - matchCatalogsTractMag17to21p5
-      - matchCatalogsTractStarsSNR5to80
-      - matchCatalogsTractGxsSNR5to80
-      - PA1
-      - PF1_design
-      - AM1
-      - AM2
-      - AM3
-      - AD1_design
-      - AD2_design
-      - AD3_design
-      - AF1_design
-      - AF2_design
-      - AF3_design
-      - AB1
-      - modelPhotRepGal1
-      - modelPhotRepGal2
-      - modelPhotRepGal3
-      - modelPhotRepGal4
-      - modelPhotRepStar1
-      - modelPhotRepStar2
-      - modelPhotRepStar3
-      - modelPhotRepStar4
-      - psfPhotRepStar1
-      - psfPhotRepStar2
-      - psfPhotRepStar3
-      - psfPhotRepStar4
-      # tract-level on coadd products
-      - TE1
-      - TE2
-      - wPerp
-      - skyObjectMean
-      - skyObjectStd
-    description: |
-      Set of tasks for calculation of metrics via faro.
-      These tasks are a mix of visit- and tract-level.
-
-      Tasks that require single-frame products use Calibrated Source Tables,
-      which are available after consolidateSourceTable (step2).
-      Tasks that require coadd products use Object Tables which are available
-      after consolidateObjectTable (step3).
+  subsets:
```

Comment:
You've added an extra `subsets:` header here (there's also one way earlier, since this file also defines all the `stepN` subsets).  You'll want to delete this one and then de-dent everything below it.

Reply:
Should I keep the one way earlier or is that supposed to be there?

Reply:
The one way earlier should stay as-is; just drop this one.